src/main/flight/mixer.h

```diff
@@ -199,6 +200,8 @@ void filterServos(void);
 extern int16_t motor[MAX_SUPPORTED_MOTORS];
 extern int16_t motor_disarmed[MAX_SUPPORTED_MOTORS];
 
+bool motorLimitReached;
```

Comment:
Technically C allows this (C++ doesn't), but it is poor practice. 
At least it is better than extern declaration in C file (the types should not get out of sync)